### PR TITLE
fix: name of gld purchases function to fit name in vignette

### PR DESCRIPTION
**Pull request by signekb**

### CODE REVIEW

File: vignettes/images/function-flow.puml [hunk 1]
```diff
@@ -47,7 +47,7 @@ card classify_diabetes_status() as cd {
         action "include_hba1c()" as in_hba1c <<inclusion>>
         action "include_diabetes_diagnosis()" as in_diagnosis <<inclusion>>
         action "include_podiatrist_services()" as in_podiatrist <<inclusion>>
-        action "include_purchases_gld()" as in_gld <<inclusion>>
```
Comment (lwjohnst86) [inline vignettes/images/function-flow.puml:50]: hmm, but now the `exclude_purchases_of_weight_loss_drugs()` doesn't match.

Comment (signekb) [reply vignettes/images/function-flow.puml:50]: We could rename that to `exclude_weight_loss_drug_purchases` or is that too crazy?

Comment (lwjohnst86) [reply vignettes/images/function-flow.puml:50]: Yea, that would work, since we use the abbreviation for GLD anyway. Good!

Comment (signekb) [reply vignettes/images/function-flow.puml:50]: I have changed `weight_loss_drugs` to `wld` in #75

**Comment by signekb** [comment]
I have added this to #75 instead.

**Pull request closed without merging**